docs/tracer/README.md

```diff
@@ -0,0 +1,39 @@
+# Cairo Tracer
+
+Cairo-rs offers a tracer which gives you a visualization of how your memory and registers change line after line as the VM executes the code.
```

Comment:
We should replace cairo-rs with cairo-vm that is the new name

Reply:
fixed